home.html

```diff
@@ -0,0 +1,141 @@
+                    <i class="fas fa-shopping-cart cart-icon">
+                        <div class="ecom-badge">5</div>
+                        <span class="cart-text">cart</span></i>
+                </a>
+            </div>
+        </nav>
+      </header>
+      <main class="ecom-main px-5 py-3">
+          <div class="home-categories">
+              <div class="categoies p-1">
+                  <a href="" class="category-links flex-col-center">
+                      <img src="/assets/IMG_20220205_233718.png" alt="" class="category-img">
+                      <p class="category-text p-1">Mobiles</p>
+                  </a>
+              </div>
+              <div class="categoies p-1">
+                <a href="" class="category-links flex-col-center">
+                    <img src="/assets/fashion.jpeg" alt="" class="category-img">
+                    <p class="category-text p-1">Fashion</p>
+                </a>
+            </div>
+            <div class="categoies p-1">
+                <a href="" class="category-links flex-col-center">
+                    <img src="/assets/headphone-1.jpeg" alt="" class="category-img">
+                    <p class="category-text p-1">Electronics</p>
+                </a>
+            </div>
+            <div class="categoies p-1">
+                <a href="" class="category-links flex-col-center">
+                    <img src="/assets/home.png" alt="" class="category-img">
+                    <p class="category-text p-1">Home</p>
+                </a>
+            </div>
+            <div class="categoies p-1">
+                <a href="" class="category-links flex-col-center">
+                    <img src="/assets/appliances_1.jpg" alt="" class="category-img">
+                    <p class="category-text p-1">Appliances</p>
+                </a>
+            </div>
+          </div>
+          <section class="ecom-banner">
+              <img src="/assets/E-commerce-banner.jpg" alt="" class="banner-img my-2">
+          </section>
+          <div class="bottom-card">
+              <a href="" class="bottom-card-links">
+                <div class="card">
+                    <div class="card-main-section">
+                        <img src="/assets/IMG_20220205_233718.png" class="product-img" alt="">
+                        <main class="middle flex-col-center">
+                            <!-- <p class="para">Apple</p> -->
+                            <div class="small-1">Apple series</div>
+                            <p>Best Deals</p>
+                            <div>from 17900/-</div>
+                        </main>
+                    </div>
+                </div>
+              </a>
+              <a href="" class="bottom-card-links">
+                <div class="card">
+                    <div class="card-main-section">
+                        <img src="/assets/IMG_20220205_233718.png" class="product-img" alt="">
```

Comment:
image name should be meaningful . like - "product-imgname" . easy to understand .